client/src/Components/Dialogs/CreateTourDialog.jsx

```diff
@@ -41,6 +41,19 @@ class CreateTourDialog extends React.Component {
         this.props.onMapTypeChanged && this.props.onMapTypeChanged({ origin: this, mapType: e.target });
     }
 
+    _translateType(type) {
+        const { messages, formatMessage } = this.props.intl;
```

Comment:
We should store in mongo db instead of 'Earth', 'Image' we should store numbers numbers (1 - Earth, 2 - Image). And pass to front end available types of tour (Right now it is hard coded in frontend).
![image](https://user-images.githubusercontent.com/9947582/57519940-2566f780-7325-11e9-8961-623fbf6de2fe.png)